### ISSUE TITLE
ERROR with PG15:  could not load library "pllua.so": undefined symbol: parse_variable_parameters

### ISSUE THREAD

**Issue by df7cb** [pllua/pllua#10]
Hi,

pllua does not work with PG15:

```
21:36:24 --- /<<PKGBUILDDIR>>/expected/pllua.out	2021-06-05 01:29:02.000000000 +0000
21:36:24 +++ /<<PKGBUILDDIR>>/results/pllua.out	2022-05-19 19:36:22.726445789 +0000
21:36:24 @@ -1,270 +1,142 @@
21:36:24  --
21:36:24  CREATE EXTENSION pllua;
21:36:24 +ERROR:  could not load library "/<<PKGBUILDDIR>>/debian/postgresql-15-pllua/usr/lib/postgresql/15/lib/pllua.so": /<<PKGBUILDDIR>>/debian/postgresql-15-pllua/usr/lib/postgresql/15/lib/pllua.so: undefined symbol: parse_variable_parameters
21:36:24  CREATE EXTENSION plluau;
21:36:24 +ERROR:  could not load library "/<<PKGBUILDDIR>>/debian/postgresql-15-pllua/usr/lib/postgresql/15/lib/pllua.so": /<<PKGBUILDDIR>>/debian/postgresql-15-pllua/usr/lib/postgresql/15/lib/pllua.so: undefined symbol: parse_variable_parameters
```

**Comment by RhodiumToad** [comment]
Time to go find out what changed...

**Comment by gsiz007** [comment]
Hi,

I found this change on PG15.

https://git.postgresql.org/gitweb/?p=postgresql.git;a=commitdiff;h=791b1b71da35d9d4264f72a87e4078b85a2fcfb4

parse_fixed_parameters() -> setup_parse_fixed_parameters()
parse_variable_parameters() -> setup_parse_variable_parameters()

**Comment by RhodiumToad** [comment]
Closed by f42cad4.
